src/SFA.DAS.ApplyService.InternalApi/Services/GatewayApiChecksService.cs

```diff
@@ -110,7 +110,7 @@ private async Task LookupCharityCommissionDetails(ApplyGatewayDetails applyGatew
                 var message =
                     $"Unable to parse charity registration number from charity number in ukrlp: '{charityNumberFromUkrlp}'";
                 _logger.LogError(message);
```

Comment:
this is no longer an error, but an occasional, but expected condition.  Can you change .LogError to .LogInformation

Reply:
Perhaps change the message as well. To something like `Charity registration number cannot be parsed into a number, skipping getting details from charity commission for {charityNumber}`

Reply:
Done